eventmesh-protocol-plugin/eventmesh-protocol-meshmessage/src/main/java/org/apache/eventmesh/protocol/meshmessage/MeshMessageProtocolAdaptor.java

```diff
@@ -108,31 +107,33 @@ public List<CloudEvent> toBatchCloudEvent(ProtocolTransportObject protocol) thro
     public ProtocolTransportObject fromCloudEvent(CloudEvent cloudEvent) throws ProtocolHandleException {
         validateCloudEvent(cloudEvent);
         String protocolDesc =
-            cloudEvent.getExtension(Constants.PROTOCOL_DESC) == null ? null : cloudEvent.getExtension(Constants.PROTOCOL_DESC).toString();
-
-        if (StringUtils.equals(MeshMessageProtocolConstant.PROTOCOL_DESC_HTTP, protocolDesc)) {
-            HttpCommand httpCommand = new HttpCommand();
-            Body body = new Body() {
-                final Map<String, Object> map = new HashMap<>();
-
-                @Override
-                public Map<String, Object> toMap() {
-                    if (cloudEvent.getData() == null) {
+                cloudEvent.getExtension(Constants.PROTOCOL_DESC) == null ? null : cloudEvent.getExtension(Constants.PROTOCOL_DESC).toString();
+
+        switch (Objects.requireNonNull(protocolDesc)) {
+            case MeshMessageProtocolConstant.PROTOCOL_DESC_HTTP:
+                HttpCommand httpCommand = new HttpCommand();
+                Body body = new Body() {
+                    final Map<String, Object> map = new HashMap<>();
+
+                    @Override
+                    public Map<String, Object> toMap() {
+                        if (cloudEvent.getData() == null) {
+                            return map;
+                        }
+                        map.put(MeshMessageProtocolConstant.PROTOCOL_KEY_CONTENT, new String(cloudEvent.getData().toBytes(),
+                                Constants.DEFAULT_CHARSET));
```

Comment:
Consider putting the second argument on the same line.

Reply:
For this line CheckStyle gave me a warning saying there are more words in the same line, and the line needs to be separated.

Reply:
The second argument is on the same line but when I tried to put all the 3 arguments in the same line, I got the warning as shown below:
<img width="320" alt="image" src="https://github.com/apache/eventmesh/assets/124816912/acd33f5d-66eb-4e58-ac9b-dbe86cfece39">

Reply:
Like this:
```java
map.put(MeshMessageProtocolConstant.PROTOCOL_KEY_CONTENT,
               new String(cloudEvent.getData().toBytes(), Constants.DEFAULT_CHARSET));
```
Because `new String(cloudEvent.getData().toBytes(), Constants.DEFAULT_CHARSET)` is the second param of `map.put`.

Reply:
I have put the whole second argument in the next line as suggested, thank you for the info :)